
<launch>
            <arg name="scan_topic"       default="zed/scan"/>
            <arg name="cloud_frame_id"   default="zed_camera_center"/>
            <node name="depthimage_to_laserscan" pkg="depthimage_to_laserscan" type="depthimage_to_laserscan" output="screen">
                <param name="scan_height" value="7"/>
                <param name="output_frame_id" value="$(arg cloud_frame_id)"/>
                <param name="range_min" value="0.1"/>
                <remap from="image" to="depth/depth_registered"/>
                <remap from="camera_info" to="depth/camera_info"/>
                <remap from="scan" to="$(arg scan_topic)"/>
            </node>
</launch>
